tenable/io/v3/base/schema/explore/search.py

```diff
@@ -13,15 +13,17 @@ class SortSchema(Schema):
     Schema for the sorting sub-object
     '''
     property = marshm_fields.Str()
-    order = marshm_fields.Str(validates=v.OneOf(['asc', 'desc']))
+    order = marshm_fields.Str(validate=v.OneOf(['asc', 'desc']))
 
 
 class SearchSchema(Schema):
     '''
     Schema supporting the search request
     '''
-    fields = marshm_fields.List(marshm_fields.Str())
-    filter = marshm_fields.Nested(FilterSchema)
-    limit = marshm_fields.Int()
-    next = marshm_fields.Int()
-    sort = marshm_fields.List(marshm_fields.Nested(SortSchema))
+    fields = marshm_fields.List(marshm_fields.Str(), allow_none=True,
+                                default=None)
+    filter = marshm_fields.Nested(FilterSchema, allow_none=True, default=None)
+    limit = marshm_fields.Int(default=1000)
+    next = marshm_fields.Str(allow_none=True)
+    sort = marshm_fields.List(marshm_fields.Dict(), allow_none=True,
```

Comment:
why are marshmallow fields being imported as "marshmallow_fields"?  This is inconsistent with other code already in pyTenable mainline.

Reply:
if we import marshmallow fields in its original form then it will create a conflict with the available fields parameter for schema validation.